doc/source/report/ultradian.rst

```diff
@@ -127,27 +124,59 @@ Basic steps::
 
     4. Determine the "random" term
 
-Longitudinal data analysis.
+.. plot:: report/plots/plot_24H_seasonal_AS.py
+
+   Seasonal variation of AS probability (circadian).
+
+**************************
+Longitudinal data analysis
+**************************
 
+Description
+^^^^^^^^^^^
 -  Attempts for mixed models
 
-   -  The mixed model is frequently used for longitudinal analysis.
-      We should specify the random effects and fixed effects first,
-      such as the IS/AS condition, the purpose of the active: like
-      food or water and any other important features. Then we can use
-      linear mixed model to get the pattern of the movements in
-      different time period.
-   -  Statsmodels.api package can be used to decompose the dataset we
-      have.
+   The mixed model is frequently used for longitudinal analysis. We should specify the random effects and fixed effects first. Since it is ultradian analysis so we only need to focus on the hour factor and their cycle which we can get from the previous LS test. The random effect is the mouse id. Basically we have 4 different mouses in one strain and we only want to compare the different pattern among these three strains. So if we set the random effect to be mouse id, the effects from different mouses will be cancelled off and we can also test the significance of these effects. The response variable will be one of the six features listed before. After that we can use the mixed model to get the pattern of the movements in different time period.
```

Comment:
"The mixed model is frequently used for longitudinal analysis. We should specify the random effects and fixed effects first. Since it is ultradian analysis, we only need to focus on the hour factor and their cycle which we can get from the previous LS test. The random effect is the mouse id. We have four different mice in one strain and only want to compare the different patterns among these three strains. So if we set the random effect to be mouse id, the effects from different mouses will be cancelled out and we can also test the significance of these effects. The response variable will be one of the six features listed before. After that we can use the mixed model to get the pattern of the movements in different time period."